SmilesMiles/SmilesMilesApp.swift

```diff
@@ -6,12 +6,21 @@
 //
 
 import SwiftUI
+import GoogleSignIn
 
 @main
 struct SmilesMilesApp: App {
     var body: some Scene {
         WindowGroup {
             MainView()
+                .onOpenURL { url in
+                              GIDSignIn.sharedInstance.handle(url)
+                            }
+                .onAppear {
```

Comment:
Comment this if not required

Reply:
Thanks! Added a print statement to verify the previous sign-in.